src/spaceone/repository/service/plugin_service.py

```diff
@@ -286,17 +287,33 @@ def list(self, params):
         """
 
         repo_mgr: RepositoryManager = self.locator.get_manager('RepositoryManager')
-        repository_id = params['repository_id']
-        repo_vo = repo_mgr.get_repository(repository_id)
-
-        plugin_mgr = self._get_plugin_manager_by_repo(repo_vo)
         query = params.get('query', {})
         only = query.get('only', [])
 
         if 'registry_url' in only:
             only.remove('registry_url')
 
-        return plugin_mgr.list_plugins(query)
+        if params.get('repository_id'):
+            repository_id = params['repository_id']
+            repo_vo = repo_mgr.get_repository(repository_id)
+
+            plugin_mgr = self._get_plugin_manager_by_repo(repo_vo)
+
+            return plugin_mgr.list_plugins(query)
+        else:
+            list_repositories = repo_mgr.list_repositories({})
+
+            list_plugin_in_all_repositories = [[], 0]
+            for repository_vo in list_repositories[0]:
```

Comment:
As in the above result, the for statement is changed as follows.

`for repository_vo in repository_vos:`